### DISCUSSION TITLE
Tandoor Recipes fails to update

### DISCUSSION THREAD

**Discussion by Wolffoxfangs** [tteck/Proxmox#2646] [Help Needed] [answered]
hey there! i tried running the bash script command and just the update command in the LXC and i continue to get this error

![image](https://github.com/tteck/Proxmox/assets/16524600/90de6163-001f-45cb-bbcc-65285ee63473)

Ive attempted an apt update and upgrade before and after running the LXC update command and it continues to fail. not sure if its a me issue or something else! 

thanks for all you do!

**Comment by Wolffoxfangs** [comment]
after the first error any reattempts to update get met with this 

![image](https://github.com/tteck/Proxmox/assets/16524600/a2093280-d910-4de8-9c31-fdf274ca4386)

It doesnt seem to break Tandoor itself, from what i can tell the front end looks and acts alright even after the errors

**Comment by tteck** [comment, marked as answer]
Modified the update function https://github.com/tteck/Proxmox/commit/e15d82fb7c9c2a4cd6988c27c8c28d12747a6a92

To update manually, copy/paste the following in the LXC console
```
export $(cat /opt/tandoor/.env | grep "^[^#]" | xargs)
cd /opt/tandoor/
pip3 install -r requirements.txt
/usr/bin/python3 /opt/tandoor/manage.py migrate
/usr/bin/python3 /opt/tandoor/manage.py collectstatic --no-input
/usr/bin/python3 /opt/tandoor/manage.py collectstatic_js_reverse
cd /opt/tandoor/vue
yarn install
yarn build
systemctl restart gunicorn_tandoor
```

**Reply by Wolffoxfangs** [reply]
Thank you so much for your speedy fix! i restored from a backup and ran the update just to make sure there wasnt any other hiccups hiding in there! I greatly appreciate your work and your speedy responses!
